DQMOffline/L1Trigger/src/L1TTauOffline.cc

```diff
@@ -704,6 +704,19 @@ void L1TTauOffline::getProbeTaus(const edm::Event& iEvent,
       TLorentzVector mytau;
       mytau.SetPtEtaPhiE(tauIt->pt(), tauIt->eta(), tauIt->phi(), tauIt->energy());
 
+      if ((*antimu)[tauCandidate].workingPoints.empty()) {
+        LogWarning("This offline tau has no antimu discriminator, skipping");
```

Comment:
@lathomas isnt the structure with the `LogWarning` something like
```edm::LogWarning  ("category") << "message to be printed"; ```

Reply:
Yes I think this should be
```edm::LogWarning("L1TTauOffline") << "This offline tau has no antimu discriminator, skipping";```
Please do the same with the other ones too

Reply:
Ah yes, will do.

Reply:
Hi @lathomas please dont forget to push your fix :)